src/Sylius/Bundle/ImportExportBundle/Behat/ImportExportContext.php

```diff
@@ -0,0 +1,637 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Bundle\ImportExportBundle\Behat;
+
+use Behat\Behat\Hook\Scope\AfterScenarioScope;
+use Behat\Gherkin\Node\TableNode;
+use EasyCSV\Writer;
+use Sylius\Component\ImportExport\Model\ExportJobInterface;
+use Sylius\Component\ImportExport\Model\ExportProfileInterface;
+use Sylius\Component\ImportExport\Model\ImportJobInterface;
+use Sylius\Component\ImportExport\Model\ImportProfileInterface;
+use Sylius\Component\ImportExport\Model\JobInterface;
+use Sylius\Component\ImportExport\Model\ProfileInterface;
+use Sylius\Component\Resource\Repository\RepositoryInterface;
+use Symfony\Component\Process\Process;
+use Sylius\Bundle\ResourceBundle\Behat\DefaultContext;
+
+/**
+ * @author Mateusz Zalewski <mateusz.zalewski@lakion.com>
+ * @author Łukasz Chruściel <lukasz.chrusciel@lakion.com>
+ */
+class ImportExportContext extends DefaultContext
+{
+    /**
+     * @var Process
+     */
+    private $process;
+
+    /**
+     * @var string
+     */
+    private $processOutput;
+
+    /**
+     * @var string
+     */
+    private $filePath;
+
+    /**
+     * @var JobInterface
+     */
+    private $job;
+
+    /**
+     * @var ProfileInterface
+     */
+    private $profile;
+
+    /**
+     * @var string
+     */
+    private $type;
+
+    /**
+     * @param string|null $applicationName
+     */
+    public function __construct($applicationName = null)
```

Comment:
Your constructor has the same definition as parent one. As you only pass variables to parent constructor, it's not needed.